plugins/modules/content_export_library.py

```diff
@@ -116,42 +92,24 @@
     from_history_id: 12345
 '''
 
-from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloAnsibleModule, _flatten_entity
+from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloContentExportBaseModule
 
 
-class KatelloContentExportModule(KatelloAnsibleModule):
+class KatelloContentExportModule(KatelloContentExportBaseModule):
     pass
 
 
 def main():
     module = KatelloContentExportModule(
         foreman_spec=dict(
             destination_server=dict(required=False, type='str'),
```

Comment:
I've opted into not moving `destination_server` and `fail_on_missing_content` because:
1. they don't exist for `repository` exports, so we'd need another layer of sub-classing
2. the doc-strings differ for `fail_on_missing_content`, so we'd not save much on the doc-fragments